### PR TITLE
Add staking module query commands

### PR DESCRIPTION
**Pull request by PrathyushaLakkireddy**

### CODE REVIEW

File: internal/modules/staking/query.py [hunk 1]
```diff
@@ -7,23 +7,10 @@
 RPC = env.RPC
 CHAINID = env.CHAINID
 
-# 'query_staking_validators' fetches the validators information and return reponse in json format.
-def query_staking_validators():
+# `query_delegator_delegation` queries individual delegator delegations.
```
Comment (anilCSE) [inline internal/modules/staking/query.py:10]:
```suggestion
# `query_delegator_delegation` queries individual delegator's delegation associated with a validator.
```

Comment (anilCSE) [inline internal/modules/staking/query.py:10]:
```suggestion
# `query_delegator_delegation` queries individual delegator's delegation associated with a validator.
```

File: internal/modules/staking/query.py [hunk 2]
```diff
@@ -7,23 +7,10 @@
 RPC = env.RPC
 CHAINID = env.CHAINID
 
-# 'query_staking_validators' fetches the validators information and return reponse in json format.
-def query_staking_validators():
```
Comment (anilCSE) [inline internal/modules/staking/query.py:11]: did we remove this? Also just `query_validators` should suffice

Comment (anilCSE) [reply internal/modules/staking/query.py:11]: ah we have query_validator_set. makes sense

File: internal/load-test/query_load.py [hunk 3]
```diff
@@ -9,7 +9,7 @@
 from internal.core.keys import keys_show
 from internal.modules.bank.query import query_balances
 from internal.modules.staking.query import (
-    query_delegator_delegations,
+    query_delegator_delegation,
     query_staking_validators,
```
Comment (anilCSE) [inline internal/load-test/query_load.py:13]:
```suggestion
```

Comment (PrathyushaLakkireddy) [reply internal/load-test/query_load.py:13]: @anilCSE, can we replace that with `query_validator_set` method.

Comment (anilCSE) [reply internal/load-test/query_load.py:13]: Yes, let do that

**Review by anilCSE — APPROVED** [review]

**Pull request merged**